### DISCUSSION TITLE
Replication in the HRS-VBS

### DISCUSSION THREAD

**Discussion by petertanksley** [dayoonkwon/BioAge#5] [Q&A] [answered]
Hello!

I am attempting to use BioAge to project all three algorithms into the HRS (I have access to the VBS data). Following along your vignette, I am able to project all three successfully into the VBS but my results diverge from what you show (in some cases, quite substantially). You mentioned that you cleaned the VBS data prior to using it in the example. Could you describe what steps you took to clean the data (e.g., changed units of measurement, dropped cases, etc)?

Cheers!

**Comment by dayoonkwon** [comment, marked as answer]
Hello Peter,

As you can see from the vignette, I removed outliers (5 standard deviations
above and below the mean).
Make sure you match the units of each measure:
<img width="771" alt="bioage_unit" src="https://user-images.githubusercontent.com/53448379/132437530-ab2db3e4-b06e-4a0f-9dfd-90133fd64204.png">

Here is my summary of the biomarkers for your reference:
<img width="734" alt="data_summary" src="https://user-images.githubusercontent.com/53448379/132437400-274c1ff2-0891-45fd-9394-f1c881547ea0.png">

Please let me know if you have further questions.

Best,
Dayoon

**Reply by petertanksley** [reply]
Hello Dayoon!

Thanks for getting back to me. I think I understand you correctly. Let me see. You mentioned in the vignette (prior to loading the VBS data) that you had previously downloaded and cleaned the data. Reading your response though, it sounds like the only data cleaning that occurred were those manipulations listed in your vignette (adjusted the coding of the gender variable, units of measurement, etc.), have I got that right? Otherwise, the VBS data were untouched prior to those calculations? 

Also, was the race variable coded in any specific way (e.g., was it combined with an indicator of hispanicity)? Lastly, I am afraid your response might have got a little scrambled  somehow. The list of biomarkers and units of measurement all appear in one column (confusing things a little) and I cannot view the screenshot you posted. This is what I am seeing (I hope you can see it on your end!):

![Screen Shot 2021-09-04 at 9 49 39 PM](https://user-images.githubusercontent.com/54998131/132113230-7ad92945-d33b-4e47-a56a-c29528922d37.jpg)

Thanks so much for your time!
Cheers,
Peter

**Reply by dayoonkwon** [reply]
Hello Peter,

My bad, I hope you can see the images now. I received the data from @danbelsky and then proceed with the data cleaning mentioned in the vignette. @danbelsky can explain in detail if he further cleaned the data after downloading it. 

Also, race is coded as 1) white/caucasian 2) black/african american 3) others.  

Best,
Dayoon
